hack/usoc23: Add session 03 - Behind the scenes (#275)
unikraft/docs: content/en/community/hackathons/sessions/behind-scenes/content/Work-Items_02.-KVM-x86_64.md

[    0.000000] Info: [libkvmplat] <bootinfo.c @   62> Boot loader : qemu-multiboot                                                                    
[    0.000000] Info: [libkvmplat] <bootinfo.c @   67> Command line: ./build/app-helloworld_qemu-x86_64
[...]
[    0.000000] Info: [libukboot] <boot.c @  288> Initialize memory allocator...
[    0.000000] Info: [libukallocbbuddy] <bbuddy.c @  515> Initialize binary buddy allocator 183000
[    0.000000] Info: [libukboot] <boot.c @  313> Initialize IRQ subsystem...
[    0.000000] Info: [libukboot] <boot.c @  320> Initialize platform time...
[    0.000000] Info: [libkvmplat] <tscclock.c @  255> Calibrating TSC clock against i8254 timer
[...]
[    0.111088] Info: [libukboot] <boot.c @  369> Pre-init table at 0x120060 - 0x120060
[    0.111823] Info: [libukboot] <boot.c @  380> Constructor table at 0x120060 - 0x120060
[    0.112619] Info: [libukboot] <boot.c @  401> Calling main(1, ['./build/app-helloworld_qemu-x86_64'])
Hello world
[    0.114181] Info: [libukboot] <boot.c @  410> main returned 0, halting system
[    0.115266] Info: [libkvmplat] <shutdown.c @   35> Unikraft halted
```

> We now see a couple of information about what happens from the booting process until the application actually starts.

Reviewer: ```suggestion
We now see some information about what happens from the booting process until the application actually starts.
```
